superset-frontend/src/explore/components/controls/DateFilterControl.jsx

```diff
@@ -396,7 +396,7 @@ export default class DateFilterControl extends React.Component {
       );
     });
     return (
-      <Popover id="filter-popover" placement="top" positionTop={0}>
+      <Popover id="filter-popover" placement="left">
```

Comment:
why change this?

Reply:
make the pop up flow on the left of the botton, so the pop up is shown properly.
before:
![image](https://user-images.githubusercontent.com/24424690/88363525-f1a1b180-cdb2-11ea-9e65-2a5e77dd651e.png)


after:
![image](https://user-images.githubusercontent.com/24424690/88363603-2e6da880-cdb3-11ea-852a-b98a1ffcc080.png)